python/cuspatial/cuspatial/tests/test_align.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 NVIDIA CORPORATION.
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+from shapely.geometry import Polygon
+
+import cuspatial
+
+
+def test_align():
+    gpdpdf = gpd.GeoSeries(
+        [
+            Polygon(((-8, -8), (-8, 8), (8, 8), (8, -8))),
+            Polygon(((-2, -2), (-2, 2), (2, 2), (2, -2))),
+        ]
+    )
+    gpdshort = gpdpdf.iloc[0:1]
+    pdf = cuspatial.from_geopandas(gpdpdf)
+    short = cuspatial.from_geopandas(gpdshort)
+    gpdaligned = gpdshort.align(gpdpdf)
+    shortaligned = short.align(pdf)
```

Comment:
Can we rename the variable according to our convention? gpdaligned is the result computed from geopandas so should be `expected`, the other one is computed from cuspatial so should be `got`. And I suggest computing `expected` and `got` separately. It's hard to read atm. This applies to all following tests. Thanks.

Reply:
Good idea, I like got and expected.